test/over_react/util/handler_chain_util_test.dart

```diff
@@ -158,7 +160,7 @@ main() {
             test('calls all functions in order', () {
               var calls = [];
 
-              var functions = new List.generate(5, (index) {
+              List<Function> functions = new List.generate(5, (index) {
```

Comment:
#supernit for something like this, I think it might be preferred to do
```dart
var functions = new List<Function>.generate(5, (index) {
```
@evanweible-wf ?